src/main.cpp

```diff
@@ -6794,6 +6793,107 @@ ThresholdState VersionBitsTipState(const Consensus::Params& params, Consensus::D
     return VersionBitsState(chainActive.Tip(), params, pos, versionbitscache);
 }
 
+bool LoadMempool(void)
+{
+    FILE* filestr = fopen((GetDataDir() / "mempool.dat").c_str(), "r");
```

Comment:
restrict means the argument is assumed not to alias any other parameter, I think?